### ISSUE TITLE
-a does not work but --add does

### ISSUE THREAD

**Issue by MoOx** [pwmckenna/node-travis-encrypt#22]
```console
~/Sync/Development/statinamic master* ⇣
❯ travis-encrypt -a -r MoOx/statinamic GITHUB_TOKEN=b...LALALA...7
# GITHUB_TOKEN
Rru/KR1...LALALA...2mA/760=

~/Sync/Development/statinamic master* ⇣
❯ travis-encrypt --add --repo MoOx/statinamic GITHUB_TOKEN=b...LALALA...7
Wrote 1 blob(s) to .travis.yml
```

Is this on purpose?

**Comment by rexxars** [comment]
No, this was a bug. Fixed and released as 2.1.1. Thanks for bringing this to our attention!

**Comment by MoOx** [comment]
Thanks!
